login.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+
+<link rel="stylesheet" type="text/css" href="css/login.css">
+
+</head>
+<body>
+
+<div id="external">
+<div id="login">
+	<h4>Interview Feedback</h4>
+	<form method="POST">
+		<input type="text" name="userName" placeholder="User Name"> <br>
+		<input type="password" name="pw" placeholder="Password"> <br>
+		<input type="submit" name="login" value="Login">
+	</form>
+</div>
+</div>
+
+<div id="fb1" class:"clearfix">
+    <img src="assets/newfeedback.jpg">
+    <h1>Submit new feedback</h1>
+    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
+</div>
+
+<div id="fb2" class:"clearfix">
+    <img src="assets/results.jpg">
```

Comment:
Any particular reason for this relative resource URL? Why relative and not absolute? Could you describe what the difference between the two is?